src/keys/drivers/YubiKey.cpp

```diff
@@ -132,27 +132,17 @@ void YubiKey::detect()
 {
     bool found = false;
 
-    if (init()) {
```

Comment:
Why are we cutting out this check?

Reply:
@droidmonkey it's done in `isBlocking` now.